src/test/java/ru/jma/userservice/UserControllerTest.java

```diff
@@ -0,0 +1,57 @@
+package ru.jma.userservice;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.reactive.WebFluxTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.reactive.server.WebTestClient;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+import ru.jma.userservice.controller.UserController;
+import ru.jma.userservice.model.User;
+import ru.jma.userservice.service.UserService;
+
+@WebFluxTest(UserController.class)
+public class UserControllerTest {
+
+    @Autowired
+    WebTestClient webTestClient;
+
+    @MockBean
+    UserService userService;
+
+    @Test
+    void testGetUser() {
+        User user = new User((long) 1, "user", "1234");
+        User user2 = new User((long) 2, "user2", "5678");
+        Flux<User> userFlux = Flux.just(user, user2);
+        Mockito.when(userService.findAll()).thenReturn(userFlux);
```

Comment:
Add check array size

Reply:
The WebTestClient interface does not contain a hasSize() method unlike the MockMvc Class

*WebTestClient: https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/test/web/reactive/server/WebTestClient.html
*MockMvc: https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/test/web/servlet/MockMvc.html